astrokat/observe_main.py

```diff
@@ -356,7 +357,9 @@ def run_observation(opts, kat):
                 if (name == cat_tgt.name and
                         tags == ' '.join(cat_tgt.tags)):
                     tgt['target'] = cat_tgt
+                    obs_tags.extend(cat_tgt.tags)
                     break
+        cal_tags = np.unique([tag for tag in obs_tags if 'cal' in tag])
```

Comment:
My standard assumption is that calibrators will have the "cal" substring somewhere in the tag. I try not to make any assumption beyond that, because I do not know if the position of it will change, there is no clear definition on how the astronomers want this.
History shows a format `tag=<caltype>+"cal"`
But I do not know if I should enforce that, and if it is simply convenient at the moment.....

So I assume we could make an executive decision now?
I am open to suggestions...

Reply:
I like `tag=<caltype>+"cal"`.  Keeps things consistent, and is still very flexible.

Who consumes the calibration tags?  SDP pipeline?  Maybe @ludwigschwardt has a strong opinion.

Reply:
The SDP pipeline consumes them (and soon katdal too). We are getting to the point where they can be formalised. It is a good assumption that the tag will end in `cal`.

Reply:
Implemented the format `tag=<caltype>+"cal"`

If anything happens in the future that breaks this assumption, it can be dealt with then